apis/r/src/riterator.cpp

```diff
@@ -147,13 +147,17 @@ Rcpp::XPtr<tdbs::SOMAReader> sr_setup(Rcpp::XPtr<tiledb::Context> ctx,
     Rcpp::XPtr<tdbs::SOMAReader> xptr = make_xptr<tdbs::SOMAReader>(ptr);
     return xptr;
 }
+
 //' @rdname sr_setup
 //' @export
 // [[Rcpp::export]]
 bool sr_complete(Rcpp::XPtr<tdbs::SOMAReader> sr) {
    check_xptr_tag<tdbs::SOMAReader>(sr);
-   spdl::info("[sr_complete] Complete test is {}", sr->is_complete());
-   return sr->is_complete();
+   size_t nobs = sr->total_num_cells();
+   bool complt = sr->is_complete(true);
```

Comment:
total nit but can we just say ... `complete` and `result`? what are we optimizing for by leaving letters out?

Reply:
I will defer to @gspowley here -- these are existing (internal) accessor so changing them now may be feasible but we _really_ have to be sure it is worth any change and possible disruption.

Reply:
Or if you refer to just the left-hand side and the variable name: Sure. I am mostly proud I moved on from `v`.

Reply:
i'm just talking about local variable names & this is a nit

Reply:
@eddelbuettel please let's forget about it. it's a nit. not worth the sentences we've already expended on it. i regret even mentioning it.